src/PointwiseFunctions/Hydro/QuadrupoleFormula.hpp

```diff
@@ -0,0 +1,109 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/GrMhd/ValenciaDivClean/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/Hydro/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace gsl {
+template <typename>
+struct not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace hydro {
+
+namespace Tags {
+/// Tag containing the quadrupole moment, using
+/// TildeD\f$* x^ix^j\f$ (equation 21 of \cite Shibata2003),
```

Comment:
Is there a reason to not use `\f$\tilde{D} x^i x^j\f$` in tex form and instead write it out? The tex seems like would be easier to read. We also avoid `*` in tex for multiplication because it just looks kind of funny and sometimes has other meanings, like the dual of a tensor. I'd also suggest writing out that `\tilde{D}` is `\sqrt{\gamma}\rho W`, just to make life easier for future readers.

3 more occurrences below